include/SFML/Graphics/RenderStates.hpp

```diff
@@ -103,6 +113,22 @@ class SFML_GRAPHICS_API RenderStates
     ////////////////////////////////////////////////////////////
     RenderStates(const BlendMode& theBlendMode, const Transform& theTransform, const Texture* theTexture, const Shader* theShader);
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Construct a set of render states with all its attributes
+    ///
+    /// \param theBlendMode       Blend mode to use
+    /// \param theStencilMode     Stencil mode to use
+    /// \param theTransform       Transform to use
+    /// \param theTexture         Texture to use
+    /// \param theShader          Shader to use
+    ///
+    ////////////////////////////////////////////////////////////
+    RenderStates(const BlendMode&   theBlendMode,
+                 const StencilMode& theStencilMode,
+                 const Transform&   theTransform,
+                 const Texture*     theTexture,
+                 const Shader*      theShader);
+
```

Comment:
I suppose this constructor was added to be API compatible during the SFML 2 lifecycle. Because we're breaking the API, can we instead just add this extra parameter to the existing 4-parameter constructor?